changelog.md

```diff
@@ -1,3 +1,7 @@
 # Changelog
 
 ## Unreleased
+
+### Fixed
```

Comment:
Let's update this to remove the changelog entry

Reply:
Ok, I force-pushed a head that doesn't have the changelog